main_windows.go

```diff
@@ -40,9 +43,57 @@ func checkUseInteractiveMode() (bool, error) {
 	return !isWindowsService, nil
 }
 
-func runService(params otelcol.CollectorSettings) error {
+func runService(buildInfo component.BuildInfo) error {
+	configContents := `yaml:
+receivers:
+  otlp:
+    protocols:
+      http:
+        endpoint: localhost:4318
+      grpc:
+        endpoint: localhost:4317
```

Comment:
I'm unfamiliar with running things as a service on windows, but do we want to be able to customize ports there too? or is hard-coding them in this code-path an intentional choice?

Reply:
It's intentional to the best of my understanding, and I am sure someone will correct me if I got it wrong, but command is not run on this path on the autogenerated code, so I went based on that,